_posts/2018-03-01-core-team-announcement.md

```diff
@@ -8,21 +8,20 @@ author: The Core Team
 
 The year 2018 is well underway, with many exciting new technologies on the horizon for Monero. Technology is not the only thing that needs to adapt if Monero is to stay at the forefront of privacy and blockchain technologies, however. With that in mind, the Core Team would like to remind the community about their role in regards to Monero followed by an announcement regarding the membership of the Core Team. 
 
-The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
+The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community without using a mixnet, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.

```

Comment:
I think this an excellent clarification!

Reply:
This is making me think about a devising a new multi-homing scheme.